---
geometry_msgs/Pose[] candidates
float32[] distances
bool is_succeeded